partner_base_multicompany_default_queue/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Camptocamp SA
```

Comment:
I would rename it to `*_multicompany_queue` as you might add other feats other than setting defaults.

Reply:
Nerver mind, I see where it comes from.